bigchaindb/consensus.py

```diff
@@ -139,12 +139,21 @@ def validate_transaction(bigchain, transaction):
                 raise exceptions.OperationError(
                     'Only federation nodes can use the operation `CREATE`')
 
+            # validate digital asset
+            # A create transaction will have only one output
+            asset = transaction['transaction']['asset']
+            assets.validate_asset_creation(asset['data'], asset['divisible'], asset['updatable'],
+                                           asset['refillable'], transaction['transaction']['conditions'][0]['amount'])
+            # TODO: Validate uuid?
```

Comment:
If you have several nodes and/or the client signing the creation then it would be impossible for a node to change the transaction. Worst case scenario he could decide to just drop the transaction and not write it to the backlog but that is another problem